sgx-sys/src/lib.rs

```diff
@@ -76,6 +76,8 @@ mod test {
     fn builder_new() {
         let contents = sig::Contents::new(
```

Comment:
This could possibly use a comment to say the second `misc` and `attr` are the masks.

Reply:
I actually haven't fully understood this part of SGX yet. I didn't take the masks in the constructor before because I was being lazy. But the `selftest` code required me to differentiate between the two.